e2e/framework/testserver.go

```diff
@@ -187,7 +187,7 @@ func (t *TestServer) SecretsInit() (*InitIBFTResult, error) {
 
 	commandSlice := strings.Split(fmt.Sprintf("secrets %s", secretsInitCmd.Use), " ")
 	args = append(args, commandSlice...)
-	args = append(args, "--data-dir", t.Config.IBFTDir)
+	args = append(args, "--data-dir", filepath.Join(t.Config.IBFTDir, "tmp"))
```

Comment:
Is there any special reason?

Reply:
A few lines later, https://github.com/0xPolygon/polygon-edge/blob/70f9884642c307c03187adb82b0e7c24cb85051f/e2e/framework/testserver.go#L213-L246 we are re-initializing it in the same directory runs into this check. https://github.com/0xPolygon/polygon-edge/pull/705/files#diff-af4842623f99504a537de510a8411702cde6ff5671554f5076c9337135dbcef9R135-R141. 

Test Result: https://github.com/0xPolygon/polygon-edge/runs/8143748563?check_suite_focus=true

Correct me if I mistook the behavior, or if there's a better suggestion. Thanks!